decorators.rb

```diff
@@ -0,0 +1,28 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class CorrectName < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [ ] **Required:** You have implemented this correctly. However, you are required to name it **_Decorator_**. Kindly update this class name to avoid confusion going forward and also to conform with your naming. If you see other classes after this are named to follow this. Such ad _Capitalize**Decorator**_, etc.